deployments/ci.sh

```diff
@@ -87,7 +88,10 @@ else
 fi
 echo "Generating new testnet files..."
 container_cli="$(get_container_cli)"
+# Silence shellcheck warning on 'preserve_chain_opt' being an empty string.
+# shellcheck disable=SC2086
 "$container_cli" run --user 0:0 \
+    --pull always \
```

Comment:
This one-line change addresses the out-of-sync chain data we were seeing on preview, even after resolution of #1744. Now, the remote container registry is always checked. This is especially necessary for using the "main" tag, which is frequently clobbered. (The weekly testnet tags, e.g. `038-kalyke`, are static.)